ouroboros-network/src/Ouroboros/Network/Point.hs

```diff
@@ -19,40 +19,14 @@ module Ouroboros.Network.Point
 import           GHC.Generics (Generic)
 
 import           Cardano.Prelude (NoUnexpectedThunks)
-
+import           Cardano.Slotting.Slot
 import           Codec.Serialise (Serialise (..))
 
-data WithOrigin t = Origin | At !t
-  deriving (Eq, Ord, Show, Generic, Functor, Foldable, Traversable,
-            NoUnexpectedThunks, Serialise)
-
 data Block slot hash = Block
   { blockPointSlot :: !slot
   , blockPointHash :: !hash
   }
   deriving (Eq, Ord, Show, Generic, NoUnexpectedThunks)
 
-at :: t -> WithOrigin t
-at = At
-
-origin :: WithOrigin t
-origin = Origin
-
 block :: slot -> hash -> WithOrigin (Block slot hash)
 block slot hash = at (Block slot hash)
```

Comment:
Is this one even used (check other repos too)? If not, then I wouldn't mind removing it.

Reply:
It is used, yes